web-gui/src/app/dialogs/task-new-dialog/task-new-dialog.component.ts

```diff
@@ -12,6 +12,9 @@ import {mergeMap, map} from 'rxjs/operators';
 import {CheckerService} from '../../service/checker.service';
 import {CheckerConfig} from '../../model/CheckerConfig';
 
+let currentDateAndOneMonthLater = new Date();
+currentDateAndOneMonthLater.setMonth(currentDateAndOneMonthLater.getMonth() + 1);
```

Comment:
Generating the date here works, but leads to the problem that it is only generated once and therefore the time does not correspond to the current time. Here you should define a function that generates and returns the date:

![grafik](https://user-images.githubusercontent.com/63304184/152792536-7ee79ae9-9a40-45a9-b0f6-23fa4f7f2bd4.png)